frontend/src/posapp/components/reports/components/reports.css

```
@@ -0,0 +1,425 @@
+.metric-card__label {
+	font-size: 0.85rem;
+	font-weight: 600;
+	text-transform: uppercase;
+	letter-spacing: 0.03em;
+}
+
+.metric-card__value {
+	font-size: 1.35rem;
+	font-weight: 700;
+	color: var(--pos-text-primary);
+	word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)